index.html

```diff
@@ -5,41 +5,218 @@
-                some of my work and experience! If you like what you see and have a project you need coded,
-                don't hesitate to contact me.</p>
-            <p class="second-paragraph">LETS CONNECT</p>
-            <ul class="social-media">
-                <li> <a href="#"><img src="./images/github.jpg" alt="github"></a> </li>
-                <li> <a href="#"><img src="./images/medium.jpg" alt="Linkedin"></a> </li>
-                <li> <a href="#"><img src="./images/twitter.jpg" alt="medium"></a> </li>
-                <li> <a href="#"><img src="./images/Vector.jpg" alt="twitter"></a> </li>
-            </ul>
-        </div>
-    </section>
+   
+
+
+        <section class="first-section">
+
+            <div class="section2">
+                <h1>I'm Ekene, <br> glad to see <br> you!</h1>
+                <p class="first-paragraph">I'm a software developer! I can help you build a product , feature or website
+                    Look through
+                    some of my work and experience! If you like what you see and have a project you need coded,
+                    don't hesitate to contact me.</p>
+                <p class="second-paragraph">LETS CONNECT</p>
+                <ul class="social-media">
+                    <li> <a href="#"><img src="./images/github.jpg" alt="github"></a> </li>
+                    <li> <a href="#"><img src="./images/medium.jpg" alt="Linkedin"></a> </li>
+                    <li> <a href="#"><img src="./images/twitter.jpg" alt="medium"></a> </li>
+                    <li> <a href="#"><img src="./images/Vector.jpg" alt="twitter"></a> </li>
+                </ul>
+            </div>
+        </section>
+
+        <!-- Hero section -->
+
+        <div class="grid-position">
+            <div class="Grid-container">
+                <section class="second-section">
+                    <img src="./images/Snapshoot-Portfolio-(3).jpg" alt="section-eader">
+                    <h2>Tonic</h2>
+
+                    <div class="to-do">
+                        <ul>
+                            <li class="canopy">CANOPY</li>
+                            <li><img src="./images/Counter.jpg" alt="seperation"></li>
+                            <li>Back End Dev</li>
+                            <li><img src="./images/Counter.jpg" alt="seperation"></li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <p class="to-do-para">A daily selection of privately personalised reads: no accounts or sign-ups
+                        required.</p>
+
+                    <div class="works">
+                        <ul>
+                            <li class="list"><a href="#">html</a></li>
+                            <li class="list"><a href="#">css</a></li>
+                            <li class="list"> <a href="#">javascript</a></li>
```

Comment:
- [ ] [OPTIONAL] You can remove the anchor `a` tag as this is only a list and not links.